explainer.md

```diff
@@ -183,7 +183,7 @@ an environment integrity attestation on the web page and send it to the web serv
 // getEnvironmentIntegrity expects a “content binding” of the request you are
 // about to make. The content binding protects against this information being
 // used for a different request.
-// The contentBinding will be concatenated with top-level domain name and hashed
+// The contentBinding will be concatenated with eTLD+1 and hashed
```

Comment:
Thank you for this suggestion! It makes a lot of sense. Can you also tighten the language around "Cross-site tracking" while you are making this change?

Reply:
We should specify that it's the eTLD+1 of the top-level frame rather than an embedded frame.